opal-core-ws/src/main/java/org/obiba/opal/web/project/ProjectResource.java

```diff
@@ -142,6 +141,21 @@ public VCFStoreResource getVCFStoreResource() {
     return resource;
   }
 
+  @GET
+  @Transactional(readOnly = true)
+  @Path("/genotypes/summary")
```

Comment:
This is not the correct entry point. The resource is not about genotypes, but about vcf-store. The specs says the REST resource should be:

GET /project/ppp/vcf-store